scvi/model/base/_base_model.py

```diff
@@ -623,7 +627,12 @@ def load(
         >>> model.get_....
         """
         load_adata = adata is None
-        _, _, device = parse_use_gpu_arg(use_gpu)
+        _, _, device = parse_device_args(
+            use_gpu=use_gpu,
+            accelerator=accelerator,
+            devices=device,
+            return_device="torch",
```

Comment:
there are certain instances where we only expect one device. should we add a flag to the parse function like `validate_single_device` ? of course multiple devices would make sense always for cpu according to how lightning handles things